### ISSUE TITLE
11.6 not working with iOS 16.1.1

### ISSUE THREAD

**Issue by dwsugden** [dgreif/ring#1110]
# Bug Report

### Describe the Bug
 The newer versions, including v11.6, does not work with iOS 16. But I'm guessing the problem is with Hoobs??? The Bridge (11.6) is installed on Hoobs and my account is linked to Ring but the devices are not showing in Hoobs. Installing the Bridge on the iPhone via the QR code works but no Ring devices showing. Installing v9.24 solves the problem with Hoobs showing the Ring devices and they also appear on the iPhone. 

### To Reproduce
 - Start Homebridge
 - Attempt to ...
 - See error

### Expected behavior
 A clear and concise description of what you expected to happen.

### Screenshots/Logs
 If applicable, add screenshots and log output to help explain your problem.

### Additional context
 Add any other context about the problem here.

### Homebridge Ring Config
Post homebridge-ring platform config  _without sensitive information_
```

HIGHLIGHT THIS TEXT AND POST CONFIG HERE
DO NOT INCLUDE YOUR REFRESH TOKEN OR EMAIL/PASSWORD (replace them with xxx)

```

### Environment

 - OS: [e.g. Raspbian]
 - Node.js: [e.g. 12.14.1] 
 - NPM: [e.g 6.13.4] 
 - homebridge-ring: [e.g 6.0.3]
 - homebridge: [e.g. 0.4.50 or N/A]
 - hoobs: [e.g. 0.4.50 or N/A]

**Comment by dgreif** [comment]
Do you see any errors in the logs?

**Comment by dwsugden** [comment]
Thank you for replying.

I have looked at the log (sorry for not doing this initially) and found the problem:

"homebridge-ring" requires a Homebridge version of >=1.4.0 which does not satisfy the current Homebridge version of 1.3.5."

Searching the Internet, it looks as though I am not the only person with this problem, despite running the latest version of HOOBS.

I have found a solution how to get HOOBS to update to the latest Homebridge version with a script using its Terminal. I will implement the fix as soon as I get home from work.

On the applet review I will mention this problem since it looks as though many are reporting the current version of your applet not working. and having to roll back to v9.24. I suspect they have the same problem as me. This is what I think is holding back your star review on HOOBS being only 3/5.

Maybe it's in your interest to flag this up in your description and get users of HOOB to check whether Homebridge 1.40 or later is installed???

Kid regards,
Darren
